File: sample-app-java+push-notification/src/main/java/com/cometchat/sampleapp/java/fcm/voip/interfaces/VoIPPermissionListener.java
package com.cometchat.sampleapp.java.fcm.voip.interfaces;


import com.cometchat.sampleapp.java.fcm.voip.model.CometChatVoIPError;

public interface VoIPPermissionListener {
    void onPermissionsGranted();

    void onPermissionsDenied(CometChatVoIPError error);
}

File: sample-app-kotlin+push-notification/src/main/java/com/cometchat/sampleapp/kotlin/fcm/voip/interfaces/VoIPPermissionListener.kt
package com.cometchat.sampleapp.kotlin.fcm.voip.interfaces

import com.cometchat.sampleapp.kotlin.fcm.voip.model.CometChatVoIPError

interface VoIPPermissionListener {
    fun onPermissionsGranted()

    fun onPermissionsDenied(error: CometChatVoIPError?)
}
